dactyl-config.yml

```diff
@@ -59,6 +59,8 @@ targets:
             "tutorial-paychan.html": https://ripple.com/build/payment-channels-tutorial/
             "tutorial-escrow.html": https://ripple.com/build/escrow-tutorials/
             "tutorial-listing-xrp.html": https://ripple.com/build/listing-xrp-exchange/
+            # Jha added placeholder link_sub below - need to get the permalink from Wordpress
```

Comment:
Good idea. Let's get the real permalink, set it here, and remove this comment before merging.